charts/amazon-cloudwatch-observability/values.yaml

```diff
@@ -602,6 +608,37 @@ agent:
         }
       }
     }
+  windowsDefaultConfig:
+    {
+      "logs": {
+        "metrics_collected": {
+          "kubernetes": {
+            "enhanced_container_insights": true
+          },
+          "application_signals": { }
+        }
+      },
+      "traces": {
+        "traces_collected": {
+          "application_signals": { }
+        }
+      }
+    }
+  prometheus:
+    config:
+    targetAllocator:
```

Comment:
i think we'll have more TLS related stuff here, but those can come slightly later